libs/webb-ui-components/src/containers/ConfirmationCard/DepositConfirm.tsx

```diff
@@ -65,148 +60,169 @@ export const DepositConfirm = forwardRef<HTMLDivElement, DepositConfirmProps>(
-                titleClassName="text-mono-100 dark:text-mono-80"
-                className="text-mono-100 dark:text-mono-80"
-              />
-              <div className="flex items-center space-x-4">
-                {wrappableTokenSymbol && (
-                  <>
-                    <TokenWithAmount
-                      token1Symbol={wrappableTokenSymbol}
-                      amount={wrappingAmount}
-                    />
-                    <ArrowRight />
-                  </>
-                )}
-                <TokenWithAmount
-                  token1Symbol={fungibleTokenSymbol}
-                  amount={amount}
-                />
-              </div>
-            </div>
-          </Section>
+        <div className="space-y-4">
+          {/** Title */}
+          <div className="flex items-center justify-between p-2">
+            <Typography variant="h5" fw="bold">
+              {title}
+            </Typography>
+            <button onClick={onClose}>
+              <Close size="lg" />
+            </button>
+          </div>
 
-          {/** New spend note */}
-          <Section>
-            <div className="space-y-2">
+          {/** Transaction progress */}
+          {typeof progress === 'number' ? (
+            <div className="flex flex-col gap-3">
               <div className="flex items-center justify-between">
                 <TitleWithInfo
-                  titleComponent="h6"
-                  title="New Spend Note"
-                  info="New Spend Note"
+                  title="Status:"
                   variant="utility"
-                  titleClassName="text-mono-100 dark:text-mono-80"
-                  className="text-mono-100 dark:text-mono-80"
+                  titleClassName="text-mono-200 dark:text-mono-0"
                 />
-                <div className="flex space-x-2">
-                  <CopyWithTooltip textToCopy={note ?? ''} />
-                  <Button
+                <Chip color="blue">{txStatusMessage}</Chip>
+              </div>
+              <Progress value={progress} />
+            </div>
+          ) : null}
+
+          <WrapperSection>
+            {/** Wrapping info */}
+            <Section>
+              <div className="flex items-end justify-between gap-6">
+                <div className="flex flex-col gap-3">
+                  <TitleWithInfo
+                    title="Source Chain"
                     variant="utility"
-                    size="sm"
-                    className="p-2"
-                    onClick={onDownload}
-                  >
-                    <Download className="!fill-current" />
-                  </Button>
+                    info="Source Chain"
+                    titleClassName="text-mono-100 dark:text-mono-80"
+                    className="text-mono-100 dark:text-mono-80"
+                  />
+                  <ChainChip
+                    type={(sourceChain?.type as ChainType) ?? 'webb-dev'}
```

Comment:
Consider updating the `type` to `ChainType` instead of `string`. If the user passes a non-existing type, casting it into `ChainType` might result in a bug in the component.